tools/bin/check-duplicated-storages.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -e
+
+if ! command -v kubectl &> /dev/null; then
+  echo "Error: kubectl is not installed. Please install it first."
+  exit 1
+fi
+
+backup_storage_names=($(kubectl get backupstorage -n everest-system --no-headers -o=jsonpath='{.items[*].metadata.name}'))
+
+get_value() {
+  local map="$1"
+  local key="$2"
+  for bucket in "${buckets[@]}"; do
+    k=$(echo "$bucket" | cut -d"=" -f1)
+    if [[ $k == $key  ]]; then
+      echo "$bucket" | cut -d"=" -f2
+    fi
+  done
+}
+
+echo "Backup storages which share the same bucket, region, endpointURL:"
+found=0
+declare -a buckets
```

Comment:
switched to an array instead of the flat string bc apparently `bash` doesn't work well with "\n". The script was working well with `sh` and `zsh`, but didn't work well with `bash`